applications/nrf5340_audio/src/bluetooth/ble_audio_services.c

```diff
@@ -117,6 +127,138 @@ static void vcs_flags_cb_handler(struct bt_vcs *vcs, int err, uint8_t flags)
+/**
+ * @brief  Callback handler for MCS discover finished
+ *
+ *         This callback handler will be triggered when MCS
+ *         discovery is finished. Used by the client.
+ */
+static void mcc_discover_mcs_cb(struct bt_conn *conn, int err)
+{
+	LOG_DBG("mcc_discover_mcs_cb");
+
+	if (err) {
+		LOG_ERR("Discovery of MCS failed (%d)", err);
+	} else {
+		LOG_DBG("Discovery of MCS finished");
+	}
+}
+
+/**
+ * @brief  Callback handler for sent MCS commands
+ *
+ *         This callback will be triggered when MCS commands have been sent.
+ *         Only for debugging, used by the client.
+ */
+static void mcc_send_command_cb(struct bt_conn *conn, int err, const struct mpl_cmd *cmd)
+{
+	LOG_DBG("mcc_send_command_cb");
+
+	if (err) {
+		LOG_ERR("Command send failed (%d) - opcode: %u, param: %d", err, cmd->opcode,
+			cmd->param);
+		return;
+	}
+}
+
+/**
+ * @brief  Callback handler for received notifications.
+ *
+ *         This callback will be triggered when a notification has been received.
+ *         Only for debugging, used by the client.
+ */
+static void mcc_cmd_notification_cb(struct bt_conn *conn, int err, const struct mpl_cmd_ntf *ntf)
+{
+	LOG_DBG("mcc_cmd_ntf_cb");
+
+	if (err) {
+		LOG_ERR("Command notification error (%d) - opcode: %u, result: %u", err,
+			ntf->requested_opcode, ntf->result_code);
+		return;
+	}
+}
+
+/**
+ * @brief  Callback handler for reading media state.
+ *
+ *         This callback will be triggered when the client has asked to read
+ *         the current state of the media player.
+ */
+static void mcc_read_media_state_cb(struct bt_conn *conn, int err, uint8_t state)
+{
+	LOG_DBG("mcc_read_media_cb");
+
+	if (err) {
+		LOG_ERR("Media State read failed (%d)", err);
+		return;
+	}
+
+	media_player_state = state;
+}
+#endif
+
+#if CONFIG_BT_MCS
+/**
+ * @brief  Callback handler for received MCS commands.
+ *
+ *         This callback will be triggered when the server has receieved a
+ *         command from the client or the commander.
+ */
+static void mcs_command_recv_cb(struct media_player *plr, int err,
+				const struct mpl_cmd_ntf *cmd_ntf)
+{
+	if (err) {
+		LOG_ERR("Command failed (%d)", err);
+		return;
+	}
+
+	LOG_DBG("Received opcode: %d", cmd_ntf->requested_opcode);
+
+	if (cmd_ntf->requested_opcode == BT_MCS_OPC_PLAY) {
+		play_pause_cb(true);
+	} else if (cmd_ntf->requested_opcode == BT_MCS_OPC_PAUSE) {
+		play_pause_cb(false);
+	} else {
+		LOG_WRN("Unsupported opcode");
+	}
+}
+
+/**
+ * @brief  Callback handler for getting the current state of the media player.
+ *
+ *         This callback will be triggered when the server has asked for the
+ *         current state of its local media player.
+ */
+static void mcs_media_state_cb(struct media_player *plr, int err, uint8_t state)
+{
+	if (err) {
+		LOG_ERR("Media state failed (%d)", err);
+		return;
+	}
+
+	media_player_state = state;
+}
+
+/**
+ * @brief  Callback handler for getting a pointer to the local media player
```

Comment:
Its pretty random where periods are used in Doxygen documentation now

Reply:
Not random, simply forgot some. Added them now